docs/gdd.rst

```diff
@@ -130,6 +130,27 @@ Some effects (e.g., damage) make use of a derived "strength" stat.
 This is the ability power (AP) multiplied by one of physical attack (PA) or magical attack (MA) depending on the ability type.
 For example the damage of a magical attack would be ``AP * MA``.
 
+Job Point (JP) formula
+	Current: The running total of the power so far.
+	Value: That base value of that power's attribute.
+* Power - Absolute(Value) x10 (targets attribute x2)
+* MP Cost - Value
```

Comment:
We may want to add to the JP cost if there is no MP cost. Maybe something like `+ (10 - mp_cost)`. This is just food for thought, and not something that needs to be "resolved" for this PR.

Reply:
Hmmm...oh that's a good idea. I've already been thinking many of these powers are running a little cheap.

Reply:
Also, I didn't follow the JP totals exactly. All the HP/MP giving/removing all total to the same amount: 210. I think that numbers fine, but I prefer the even 200 instead. There's also a couple other changes like, Rally is actually only 15 JP.